docs/source/usage/parameters.rst

```diff
@@ -312,6 +312,16 @@ Lattice Elements
 
             * ``<element_name>.nslice`` (``integer``) number of slices used for the application of space charge (default: ``1``)
```

Comment:
Discussed:
- hard edge approximation
- assumption of normal entry & exit to the element